components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -313,15 +313,15 @@ function getPageBackground (p: HasImageProps) {
       right: 0;
       display: block;
       transition: opacity .5s ease-in-out;
-      ${p => !p.hasImage && !p.solidColorForBackground && css`
+      ${p => !p.hasImage && !p.colorForBackground && css`
         background: linear-gradient(
             to bottom right,
             #4D54D1,
             #A51C7B 50%,
             #EE4A37 100%);
       `};
-      ${p => !p.hasImage && p.solidColorForBackground && css`
-        background: ${p.solidColorForBackground};
+      ${p => !p.hasImage && p.colorForBackground && css`
+        background: ${p.colorForBackground};
```

Comment:
Not really important but we could get rid of the `colorForBackground` check and just have a fallback:
```suggestion
        background: ${p.colorForBackground || 'linear-gradient(to bottom right, #4D54D1, #A51C7B 50%, #EE4A37 100%)'};
```